### ISSUE TITLE
Installation error with Anaconda 2.4 (Python 3.5)

### ISSUE THREAD

**Issue by xiaoweiz** [JuliaLang/IJulia.jl#383]
OS X El Capitan + Julia 0.4
I have just upgraded to Anaconda 2.4 with Python 3.5. Then running `Pkg.add("IJulia")` gives 

```julia
julia> Pkg.add("IJulia")
INFO: Initializing package repository /Users/xiaoweiz/.julia/v0.4
INFO: Cloning METADATA from git://github.com/JuliaLang/METADATA.jl
INFO: Cloning cache of BinDeps from git://github.com/JuliaLang/BinDeps.jl.git
INFO: Cloning cache of Compat from git://github.com/JuliaLang/Compat.jl.git
INFO: Cloning cache of Conda from git://github.com/Luthaf/Conda.jl.git
INFO: Cloning cache of Homebrew from git://github.com/JuliaLang/Homebrew.jl.git
INFO: Cloning cache of IJulia from git://github.com/JuliaLang/IJulia.jl.git
INFO: Cloning cache of JSON from git://github.com/JuliaLang/JSON.jl.git
INFO: Cloning cache of Nettle from git://github.com/staticfloat/Nettle.jl.git
INFO: Cloning cache of SHA from git://github.com/staticfloat/SHA.jl.git
INFO: Cloning cache of URIParser from git://github.com/JuliaWeb/URIParser.jl.git
INFO: Cloning cache of ZMQ from git://github.com/JuliaLang/ZMQ.jl.git
INFO: Installing BinDeps v0.3.19
INFO: Installing Compat v0.7.7
INFO: Installing Conda v0.1.8
INFO: Installing Homebrew v0.1.17
INFO: Installing IJulia v1.1.8
INFO: Installing JSON v0.5.0
INFO: Installing Nettle v0.2.0
INFO: Installing SHA v0.1.2
INFO: Installing URIParser v0.1.1
INFO: Installing ZMQ v0.3.0
INFO: Building Homebrew
INFO: Precompiling module JSON...
INFO: Precompiling module URIParser...
INFO: Precompiling module SHA...
INFO: Cloning brew from https://github.com/Homebrew/homebrew.git
Cloning into '/Users/xiaoweiz/.julia/v0.4/Homebrew/deps/usr'...
remote: Counting objects: 3811, done.
remote: Compressing objects: 100% (3655/3655), done.
remote: Total 3811 (delta 39), reused 522 (delta 22), pack-reused 0
Receiving objects: 100% (3811/3811), 3.27 MiB | 328.00 KiB/s, done.
Resolving deltas: 100% (39/39), done.
Checking connectivity... done.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  258k  100  258k    0     0  58185      0  0:00:04  0:00:04 --:--:-- 58194
==> Tapping staticfloat/juliadeps
Tapped 38 formulae (105 files, 432K)
HEAD is now at 4705b23 fzf: update 0.10.9 bottle.
HEAD is now at b178725 Merge pull request #83 from staticfloat/staging
INFO: Building Nettle
==> Installing dependencies for nettle: gmp
==> Installing nettle dependency: gmp
==> Downloading https://homebrew.bintray.com/bottles/gmp-6.0.0a.el_capitan.bottl
Already downloaded: /Users/xiaoweiz/Library/Caches/Homebrew.jl/gmp-6.0.0a.el_capitan.bottle.tar.gz
==> Pouring gmp-6.0.0a.el_capitan.bottle.tar.gz
🍺  /Users/xiaoweiz/.julia/v0.4/Homebrew/deps/usr/Cellar/gmp/6.0.0a: 15 files, 3.2M
==> Installing nettle
==> Downloading https://homebrew.bintray.com/bottles/nettle-2.7.1.el_capitan.bot
Already downloaded: /Users/xiaoweiz/Library/Caches/Homebrew.jl/nettle-2.7.1.el_capitan.bottle.1.tar.gz
==> Pouring nettle-2.7.1.el_capitan.bottle.1.tar.gz
🍺  /Users/xiaoweiz/.julia/v0.4/Homebrew/deps/usr/Cellar/nettle/2.7.1: 66 files, 1.7M
To relink: brew unlink nettle && brew link nettle
INFO: Building ZMQ
INFO: Building IJulia
INFO: Precompiling module Conda...
INFO: Recompiling stale cache file /Users/xiaoweiz/.julia/lib/v0.4/Conda.ji for module Conda.
WARNING: Module BinDeps uuid did not match cache file
===============================[ ERROR: IJulia ]================================

LoadError: __precompile__(true) but require failed to create a precompiled cache file
while loading /Users/xiaoweiz/.julia/v0.4/IJulia/deps/build.jl, in expression starting on line 2

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: IJulia had build errors.

 - packages with build errors remain installed in /Users/xiaoweiz/.julia/v0.4
 - build the package(s) and all dependencies with `Pkg.build("IJulia")`
 - build a single package by running its `deps/build.jl` script

================================================================================
INFO: Package database updated
```

**Comment by stevengj** [comment]
This is JuliaLang/julia#13458.  It is fixed in Julia 0.5, and hopefully in the next 0.4 release, but for now just close Julia, re-open it, and re-run `Pkg.build("IJulia")`.

**Comment by xiaoweiz** [comment]
Thanks for the quick response! It solves the problem.
